app/src/main/res/values/strings.xml

```diff
@@ -185,4 +185,5 @@
 
     <string name="zsl_setting_title">Use ZSL in Latency mode</string>
     <string name="zsl_setting_desc">Uses Zero Shutter Lag (ZSL) in Latency mode for faster capture. Certain devices may have a buggy implementation for this.</string>
+    <string name="image_queue_full">The image queue is currently full.. Please wait for a while before taking your next set of shots.</string>
```

Comment:
Double dot typo here

Reply:
Hey @quh4gko8,

Actually that was kind of intended, although this message will most likely be replaced with something else after a few more changes are pushed to the branch. Thanks for the feedback though @quh4gko8!